tests/devices/default_tensor/test_scalability.py

```diff
@@ -126,6 +126,7 @@ def circuit():
 
         _ = qml.QNode(circuit, dev)()
 
+    @pytest.mark.usefixtures("new_opmath_only")
```

Comment:
Do you know why this fails with old opmath?

Reply:
potentially the prod?

Reply:
Oh yes! Perhaps the better way to resolve this is to define the Hamiltonian as
```
qml.Hamiltonian([1.], [reduce(lambda a, b: a @ b, (Y(i) for i in range(wires)))])
```
?

Reply:
There is a separate test case with `Tensor` right above this one.

Reply:
Oh all good then 👍